### ISSUE TITLE
Add activation function and decodes to tsai losses

### ISSUE THREAD

**Issue by oguiza** [timeseriesAI/tsai#805]
tsai losses don't apply an activation or decode function to logits during inference.
